.github/workflows/main.yml

```
@@ -0,0 +1,27 @@
+name: Ruby
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.ruby }}
+    strategy:
+      matrix:
+        ruby:
+          - '3.0.3'
+
+    steps:
+    - uses: actions/checkout@v2
```

Nitpick: should we name each step?
```suggestion
    - name: Checkout
      uses: actions/checkout@v2
```